conf/playerbots.conf.dist

```diff
@@ -1460,7 +1460,7 @@ AiPlayerbot.RandombotsWalkingRPG.InDoors = 0
 
 # Specify percent of active bots
 # The default is 10. With 10% of all bots going active or inactive each minute.
-AiPlayerbot.BotActiveAlone = 100
+AiPlayerbot.BotActiveAlone = 90
```

Comment:
Maybe we can keep this value default at 100 for now, and merge other optimization of teleport first.

Reply:
I was kinda considering that as well, i will change it :)

Reply:
I did include a sync correction of the orginal value of 100 in 'src/PlayerbotAIConfig.cpp', the conf said 100 and code said 10. Or do you prefer that change in a seperate PR aswell?